app/models/phrase.rb

```diff
@@ -1,4 +1,6 @@
 class Phrase < ApplicationRecord
   has_many :section_phrases
   has_many :sections, through: :section_phrases
+
+  validates_presence_of :expression
```

Comment:
added in validation for expressions on Phrases

Reply:
Nice this should beef up coverage even more.

Reply:
Nice Work